.gitignore

```diff
@@ -14,6 +14,7 @@ build/
 Version v*
 *.bak
 config/user-*
+bin/utils.ps1
```

Comment:
why is `utils.ps1` ignored? This means we couldn't push the file from local repo

Reply:
because it is put in place by build.ps1 at build time.

Reply:
@daxgames My bad, I had it confused with `scripts/utils.ps1` instead of `bin/utils.ps1`.